### DISCUSSION TITLE
Support postgres LISTEN/NOTIFY protocol?

### DISCUSSION THREAD

**Discussion by baseplate-admin** [celery/kombu#1999] [Ideas]
Hi,

I was wondering if it was possible to support the postgres [notify](https://www.postgresql.org/docs/current/sql-notify.html) protocol.

There's projects like [`pgqueuer`](https://github.com/janbjorge/PgQueuer) (which does what celery is doing at a small scale) that aims to do just this,

Rationales:
1. I dont need to install redis/rabbitmq.
2. I can deploy on low end servers for small to mid scale websites.

**Comment by auvipy** [comment]
we have to see how to implement or utilize that

**Reply by baseplate-admin** [reply]
ভাইয়া আমার মনে হয় `PgQueuer` ব্যবহার না করে, `kombu` এর ভিতরে Postgres LISTEN/NOTIFY যুক্ত করা ভালো। তার কারণ এতে `django`+`celery`+`postgres` stack এ `redis`/`valkey` লাগবে না। 

ছোট থেকে মাঝারি মানের website এর জন্য `postgres` খারাপ না।

আপনারা চাইলে আমি `kombu` তে `postgres` LISTEN/NOTIFY যুক্ত করার চেষ্টা করতে পারি।

**Reply by auvipy** [reply]
if you can start with a Proof of concept, we can help further
